ledger/acctupdates_test.go

```diff
@@ -1613,7 +1611,7 @@ func TestAcctUpdatesCachesInitialization(t *testing.T) {
 
 	conf = config.GetDefaultLocal()
 	au, _ = newAcctUpdates(t, ml2, conf)
-	defer au.close()
+	// au and ao are closed via ml.Close() -> ml.trackers.close()
```

Comment:
`ao` is no longer a variable.
```suggestion
	// accountUpdates and onlineAccounts are closed via: ml.Close() -> ml.trackers.close()
```